recipe-server/normandy/base/api/views.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib.auth.models import User
+
+from rest_framework.viewsets import ModelViewSet
+
+from normandy.base.api.serializers import UserSerializer
+
+
+class UserViewSet(ModelViewSet):
```

Comment:
This exposes user info publicly to anyone, including emails. We can't do that.

An `APIView` that returns data for the current user is more appropriate, that way the only user data you can access is your own.